MovieTime/MovieTime/View/Movies/MTMoviesFeedViewController.swift

```diff
@@ -0,0 +1,231 @@
+//
+//  MTMoviesFeedViewController.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 08/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import UIKit
+
+class MTMoviesFeedViewController: MTViewController {
+
+    // MARK: Outlets
+    @IBOutlet weak var collectionView: UICollectionView!
+
+    @IBOutlet var searchBarMovies: UISearchBar! {
+        didSet {
+            searchBarMovies.searchBarStyle = .minimal
+            searchBarMovies.showsCancelButton = true
+            searchBarMovies.delegate = self
+        }
+    }
+
+    @IBOutlet var viewDropDownMenu: UIView!
+    @IBOutlet weak var barButtonSettings: UIBarButtonItem!
+    @IBOutlet weak var barButtonSearch: UIBarButtonItem!
+
+    // MARK: Properties
+    fileprivate lazy var viewModel: MTMoviesFeedViewModel = {
+        return baseViewModel as? MTMoviesFeedViewModel ?? MTMoviesFeedViewModel()
+    }()
+
+    private let refreshControl = MTRefreshControl()
+    private var searchBar: UISearchBar?
+    private var searchController: UISearchController?
+
+    fileprivate let interItemSpacing: CGFloat = 10
+
+    // MARK: Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupRefreshControl()
+        setupCollectionView()
+        setupUI()
+        setupViewModel()
+    }
+
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        collectionView.reloadData()
+        super.viewWillTransition(to: size, with: coordinator)
+    }
+
+    // MARK: Methods
+    private func setupRefreshControl() {
+        refreshControl.addTarget(self, action: #selector(pullToRefresh(_:)), for: .valueChanged)
+    }
+
+    private func setupCollectionView() {
+
+        if #available(iOS 10.0, *) {
+            collectionView.refreshControl = refreshControl
+        } else {
+            collectionView.addSubview(refreshControl)
```

Comment:
in which cases in this app is this `else` block going to be executed?

Reply:
Since the minimum project version  is 10.0, this will never be executed.
But considering the growth of the application, having version 9.0 will be indispensable. Soon, this part of the application will already be developed.